app/clients/lider_client.rb

```diff
@@ -0,0 +1,47 @@
+class LiderClient < ChromeClient
+    BASE_URL = 'https://www.lider.cl/supermercado/search?Ntt='
+  
+    def products_by_query(query: nil)
```

Comment:
**[rubocop]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>[Correctable] Layout/IndentationWidth: Use 2 (not 4) spaces for indentation.